drivers/sx126x/sx126x_netdev.c

```diff
@@ -39,6 +43,19 @@
 #define SX126X_MAX_SF       LORA_SF12
 #endif
 
+static netdev_t *_dev;
+
+#if defined(CPU_FAM_STM32WL)
+void isr_subghz_radio(void)
+{
+    /* Disable NVIC to avoid ISR conflict in CPU. Please note this does not
+        clear the interrupts not disable IRQ masks in radio*/
+    NVIC_DisableIRQ(SUBGHZ_Radio_IRQn);
```

Comment:
Without this. IRQ's are continuously generated ( `SX126X_IRQ_NONE`)

Reply:
Can you explain more? This seems weird

Reply:
The CPU would just stall NVIC is left enabled.

Reply:
So just clearing the IRQ does not work? That seems a bit weird, if this is because the IRQ status needs to be read and cleared before handling couldn't `sx126x_get_and_clear_irq_status` be called here and stored to a static variable? So>

```
static sx126x_irq_mask_t _subghz_irq_mask;
void isr_subghz_radio(void)
{
    NVIC_ClearPendingIRQ(SUBGHZ_Radio_IRQn);
    netdev_trigger_event_isr(_dev);
    sx126x_get_and_clear_irq_status(dev, &_subghz_irq_mask);
    cortexm_isr_end();
    /* Disable NVIC to avoid ISR conflict in CPU. */
```

Reply:
the problem is, ` sx126x_get_and_clear_irq_status(dev, &_subghz_irq_mask);`does a blocking SPI transaction on ISR :(